docs/site/src/pages/style/icons/icons.md

```diff
@@ -2,6 +2,14 @@
 
 Material [icons](https://material.io/guidelines/style/icons.html) use geometric shapes to visually represent core ideas, capabilities, or topics.
 
+Icons are commonly found in app bars, toolbars, buttons, and lists.
+
+## Font Icons
+
+By default, an `Icon` will display one of the many [material design icons freely available](https://material.io/icons/).
```

Comment:
Should we explain how to install it in order to get the `.material-icons` class implemented?
I'm not a native speaker, but I'm having a hard time figuring out the meaning of the sentence. That sounds wrong 😬 .